### ISSUE TITLE
notmaintableonly don't work (probably)

### ISSUE THREAD

**Issue by vincentbernat** [akvorado/akvorado#489]
### Discussed in https://github.com/akvorado/akvorado/discussions/454

<div type='discussions-op-text'>

<sup>Originally posted by **knvk** February  3, 2023</sup>
I don't know if this is missconfiguration, but it seems to me that the `notmaintableonly` configuration is not working.

## Steps to reproduce:
Prebuilt binary downloaded from the releases page [here](https://github.com/akvorado/akvorado/releases/download/v1.7.1/akvorado). 
```
# /opt/akvorado version
akvorado v1.7.1
  Build date: 2023-01-27T22:03:13+0000
  Built with: go1.19.5
```

I used [default config](https://raw.githubusercontent.com/akvorado/akvorado/v1.7.1/akvorado.yaml) for testing. 
Checking sanity: 
```
# /opt/akvorado orchestrator --debug --check /opt/akvorado.yaml
# (empty output)
```
Then I add `schema` block in root config:
```
schema:
  notmaintableonly:
    - SrcAddr
```
So my final config is [here](https://pastebin.com/hGSQNZmH) (Its just default config with changes noted above)

Then I check the config again to see if it works:
```
# /opt/akvorado orchestrator --check /opt/akvorado.yaml --debug
Error: invalid configuration:
Key: 'OrchestratorConfiguration.Schema.NotMainTableOnly' Error:Field validation for 'NotMainTableOnly' failed on the 'ninterfield' tag
Key: 'OrchestratorConfiguration.Inlet[0].Schema.NotMainTableOnly' Error:Field validation for 'NotMainTableOnly' failed on the 'ninterfield' tag
Key: 'OrchestratorConfiguration.Console[0].Schema.NotMainTableOnly' Error:Field validation for 'NotMainTableOnly' failed on the 'ninterfield' tag
```
---
At first I thought was that `SrcAddr` column was intersected somewhere else, and I added this option in `console` and `inlet` blocks and force-enabled it, but nothing changed.

Also, I ve looked through source-code and did not found any related tests. They all end in `maintableonly` configuration.

As I said earlier, I'm not sure if this is a bug or misconfiguration, so I started the discussion first.

Could you confirm this, please?

Thanks in advance.

UPD
1.7.2 still affected </div>

**Comment by vincentbernat** [comment]
Sorry, @knvk, I did miss your report.

**Comment by vincentbernat** [comment]
This is fixed by 3ea98142141885149a6cbe7ee0a2be8e0c98fc83.
